cunumeric/module.py

```diff
@@ -797,6 +797,60 @@ def diag(v, k=0):
         raise ValueError("diag requires 1- or 2-D array, use diagonal instead")
 
 
+def tri(N, M=None, k=0, dtype=float, *, like=None):
+    """
+    An array with ones at and below the given diagonal and zeros elsewhere.
+
+    Parameters
+    ----------
+    N : int
+        Number of rows in the array.
+    M : int, optional
+        Number of columns in the array.
+        By default, `M` is taken equal to `N`.
+    k : int, optional
+        The sub-diagonal at and below which the array is filled.
+        `k` = 0 is the main diagonal, while `k` < 0 is below it,
+        and `k` > 0 is above.  The default is 0.
+    dtype : dtype, optional
+        Data type of the returned array.  The default is float.
+    like : array_like
+        Reference object to allow the creation of arrays which are not NumPy
+        arrays. If an array-like passed in as `like` supports the
+        `__array_function__ protocol, the result will be defined by it. In this
```

Comment:
a backquote is missing after `__array_function__`. please make sure you can generate the documentation with your changes.